### PR TITLE
add patronum to repl

### PR DESCRIPTION
**Pull request by sergeysova**

### CODE REVIEW

File: website/editor/src/evaluator/index.tsx [hunk 1]
```diff
@@ -137,6 +137,17 @@ const fetchEffectorReactSSR = createEffect({
...
 })
 
+const fetchPatronum = createEffect({
+  async handler(effector) {
+    const url = 'https://unpkg.com/patronum/patronum.cjs.js'
+    const sourceMap = `${url}.map`
+    const req = await fetch(url)
+    let text = await req.text()
+    text = text.replace(/\/\/\# sourceMappingURL\=.*$/m, `//${tag}MappingURL=${sourceMap}`)
+    return createRealm(text, `ssr.js`, {effector})
```
Comment (zerobias) [inline website/editor/src/evaluator/index.tsx:147]: `'patronum.js'`

This became a name for that module in browser devtools

File: website/editor/src/evaluator/index.tsx [hunk 2]
```diff
@@ -276,8 +290,15 @@ const removeImportsPlugin = babel => ({
         case 'effector-react/ssr':
           replaceModuleImports('effectorReactSSR', path, babel)
           break
+        case 'patronum':
+          replaceModuleImports('patronum', path, babel)
+          break;
         default:
-          path.remove()
+          // TODO: import actual module after patronum refactoring to ES Modules
```
Comment (zerobias) [inline website/editor/src/evaluator/index.tsx:297]: I think we can avoid additional modules altogether, reusing them like that is indeed a good thing

**Review by zerobias — CHANGES_REQUESTED** [review]
Please update module name from `ssr.js` to `patronum.js`

File: website/editor/src/evaluator/index.tsx (hunk 1 above)
Comment (sergeysova) [inline website/editor/src/evaluator/index.tsx:147]:
```suggestion
    return createRealm(text, `patronum.js`, {effector})
```

**Comment by zerobias** [comment]
Big thanks! Now we can creat much more advanced and neat examples for documentation 😃

**Review by zerobias — APPROVED** [review]

**Pull request merged**